src/OpenLoco/Graphics/Gfx.cpp

```diff
@@ -1446,6 +1450,7 @@ namespace OpenLoco::Gfx
             }
         }
 
+        _currentFontSpriteBase = font;
```

Comment:
Add a comment that this is returning only the final lines font

Reply:
Like this?
```suggestion
        // Note that this is always the font used in the last line.
        // TODO: refactor to pair up with each line, and to not use a global.
        _currentFontSpriteBase = font;
```

Reply:
Yup